documentation/IDTA-01001/modules/ROOT/pages/includes/IDTA-01001_ValueOnly.adoc

```diff
@@ -48,6 +48,15 @@ The following rules shall be adhered to when serializing a submodel with the Ser
 
 * Submodel elements defined in the submodel other than the ones mentioned above are not subject to serialization of that SerializationModifier.
 
+
+
+The following rules shall be adhered to when serializing a submodel element different than a submodel with the SerializationModifier _Value_:
+
+* The submodel element is serialized as an unnamed JSON object.
+* A submodel element is considered a leaf submodel element if it does not contain other submodel elements. A leaf submodel element follows the rules for the different submodel elements considered in the serialization, as described for a submodel above. If it is not a leaf element, the serialization rules must be transitively followed until the value is a leaf submodel element.
```

Comment:
> A leaf submodel element follows the rules for the different submodel elements considered in the serialization, as described for a submodel above
This is also not correct, as the section above defined a property to be serialized as `${Property/idShort}: ${Property/value}` but in fact it should just `${Property/value}`.

I think we are making things much more complicated than they are by differentiating between leaf and non-leaf elements. What is wrong about just making a "recursive" definition, e.g. 

Property is serialized as 
> ${Property/Value}

SubmodelElementCollection is serialized as
> {
   "${SubmodelElement/idShort}": valueOnly(${SubmodelElement})    // for each submodel element in the collection
}

etc?

This would cover all cases at once and reduce the complexity of this whole section drastically.